.github/workflows/check-doc-sw.yml

```diff
@@ -24,3 +24,4 @@ jobs:
       - run: which pip3 && which python3 && which pip
       - run: riscv64-unknown-elf-gcc --version
       - run: pwd && source environment && cd proj/mnv2_first && pip3 list && make -j9 software
+      - run: pwd && source environment && cd proj/mnv2_first && pip3 list && make PLATFORM=sim unit
```

Comment:
Add -j9 here, too. 

Adding PLATFORM=sim wil force all the C++ to be rebuilt and -j9 will help with that.

Reply:
Yes, made this change.   Both make's now have -j8.